upgrade/upgrade-4.0.0.php

```diff
@@ -38,7 +38,7 @@
 function upgrade_module_4_0_0($module)
 {
     $tab = new Tab();
-    $tab->active = 1;
+    $tab->active = true;
```

Comment:
Did you check this works or do we need QA? I'm not 100% legacy module system accepts this

Reply:
I will make a release just after this PR. So yes a QA will be done.
I already fixed this bad assignment in the core with PHPStan (Level 2, i believe).